Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -3611,5 +3616,10 @@ public int SelectedRowIndex
     /// </summary>
     [Parameter] public Action<DataGridBatchEditItem<TItem>, DataGridColumn<TItem>, DataGridCellStyling> BatchEditCellStyling { get; set; }
 
+    /// <summary>
+    /// Gets or sets whether the column chooser is visible.
+    /// </summary>
+    [Parameter] public bool ShowColumnChooser { get; set; }

```

Comment:
Do we also want to have an option to place it at the Start or End of DataGrid?

Reply:
Hmm It's in the pager, the pager options should work to configure it.
Also there's the PagerPosition configuration.
Should be flexible enough.

Reply:
I meant to show it here if users prefer it 

![image](https://github.com/Megabit/Blazorise/assets/900302/e615717e-e5c6-4990-bb96-f834d21a9065)

But we can always do that feature later.

Reply:
And I just told you it works with the PagerPosition feature like the others.

Reply:
**This:**
**PagerElementPosition**
![image](https://github.com/Megabit/Blazorise/assets/22283161/5a13c8dd-3520-457f-8f3e-9d86df69d725)
![image](https://github.com/Megabit/Blazorise/assets/22283161/86bcdab0-e789-4d49-9c50-d3a3599e0023)
![image](https://github.com/Megabit/Blazorise/assets/22283161/bddb74d4-f788-4149-b3b8-feeeca8fd2d9)

Reply:
Ahh, I found the option now. 

I still fear the user might want to have it at the end of the table.

![image](https://github.com/Megabit/Blazorise/assets/900302/7b9396af-5f4a-4eac-8699-a0a14085ae62)

Reply:
hmm Strange I thought that was how it worked actually.
Maybe we did not really implement the feature like that.
I'd open a separate issue to review how the `PagerElementPosition` is working. I would expect that if I set to end, that it tries to actually put it at the end just like you mentioned.